testlib/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
 	api "com.diffplug.durian:durian-core:${VER_DURIAN}"
 	api "com.diffplug.durian:durian-testlib:${VER_DURIAN}"
 	api "org.junit.jupiter:junit-jupiter:${VER_JUNIT}"
+	api "org.junit.jupiter:junit-jupiter-params:${VER_JUNIT}"
```

Comment:
Is an explicit dependency on `junit-jupiter-params` needed? I think `junit-jupiter` is an aggregator artifact that should pull in `junit-jupiter-params`: [link](https://junit.org/junit5/docs/current/user-guide/#dependency-metadata-junit-jupiter).

Reply:
Thanks for the heads-up. Could have sworn that I got a compilation error when I tried experimenting with JUnit 5 features.